packages/@react-spectrum/link/docs/Link.mdx

```diff
@@ -64,6 +57,24 @@ import {Link as RouterLink} from 'react-router-dom';
 <Link><RouterLink to="/next-page">Next Page</RouterLink></Link>
 ```
 
+If the content is plain text, it will be styled and exposed to assistive technologies as a link.
+Events will need to be handled in JavaScript with the `onPress` prop.
+Note, this will not bring up link context menus in mobile, it's recommended that you use an anchor tag or something that will render one.
```

Comment:
It's not only mobile, e.g. right click.

```suggestion
Note: this will not behave like a native link. Browser features like context menus and open in new tab will not apply.
```